package.json

```diff
@@ -109,11 +104,8 @@
     "prism-themes": "^1.9.0",
     "prismjs": "^1.29.0",
     "qrcode.vue": "^3.4.1",
-    "rollup": ">=2.0.0 <3.0.0",
-    "setimmediate": "^1.0.5",
     "slugify": "^1.6.6",
     "unzipit": "^1.4.3",
-    "vite": "*",
```

Comment:
wasn't this introduced to fix different package version between `libs/ui` and main

Reply:
`histoire` use `vite` check config file (`histoire.config.ts`)

Reply:
I think it's already built-in in the `histoire` package itself: https://github.com/histoire-dev/histoire/blob/main/packages/histoire/package.json#L89

CMIIW

I think we need `vite.config` if we want to build our vue components under `libs/ui`. Currently, since we are using `pnpm-workspaces`, we can skip the build step